src/services/__tests__/mondaycoderc-schema.test.ts

```diff
@@ -0,0 +1,33 @@
+import { mondaycodercSchema } from 'services/schemas/mondaycoderc-schema';
+
+describe('mondaycodercSchema Validation', () => {
```

Comment:
Nice. Also let's add one that throws if someone provides an unsupported engine

Reply:
added a test and a new error message for invalid runtime